### DISCUSSION TITLE
Any plans on adding Chatterino7 to Chocolatey?

### DISCUSSION THREAD

**Discussion by mmattbtw** [SevenTV/chatterino7#18] [Q&A] [answered]
The main Chatterino repo has a Chocolatey package, is there any plans on adding the fork to Choco?

**Comment by zneix** [comment, marked as answer]
tl;dr I doubt this is gonna be a thing.
There's already upstream Chatterino in choco - no idea what is their policy about forks of the already existing software. Besides I'm not a Windows user so it would be very hard to even make some progress towards deploying Chatterino7 on there.
Sooner or later 7tv features may be added to upstream so I'd rather not spend time on publishing this fork if it's gonna become obsolette eventually.

**Comment by ALazyMeme** [comment]
https://community.chocolatey.org/packages/chatterino7
Pending approval
